source/sonata_simulation.rst

```diff
@@ -96,6 +97,27 @@ example::
        "integration_method" : 2
   }
 
+Format of the electrodes_file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The format of the weights file is described below:
+
+.. table::
+
+  ============================= ======================= ========== =============== ============= ==========================================
+  Group                         Field                   Type       Shape           Requirement   Description
+  ============================= ======================= ========== =============== ============= ==========================================
+  /electrodes/{electrodename}   layer                   utf8       1               Optional      Layer of the circuit in which {electrodename} is located. If the electrode is in a region without cortical layers, then "NA". If the electrode is outside the brain, then "Outside"
+  /electrodes/{electrodename}   position                float32    3               Mandatory      Position of {electrodename} in microns, in cartesian coordinates
+  /electrodes/{electrodename}   region                  utf8       1               Optional      Region in which {electrodename} is located
+  /electrodes/{electrodename}   type                    utf8       1               Optional      Either EEG or LFP
+  /electrodes/{electrodename}   offset                  uint64     1               Optional      Column index of the coefficients for {electrodename} in /electrodes/{population_name}/{node_id}
+  /electrodes/{population_name} {node_id}               float64    N_seg x N_elec  Mandatory     Scaling factor for each segment in the corresponding neuron, in mV/nA
```

Comment:
In previous entries, the format is **/electrodes/{electrodename}**, then there is **/electrodes/{populationname}** .
The way I understand from this spec is that the same **electrodename** can affect different **populationname**, is that correct?
If that is the case, how does /electrode/{electrodename}/offset works, given that is a single value that could affect multiple populations with different node count?